src/shared/base-filesystem.c

```diff
@@ -66,20 +71,20 @@ static const BaseFilesystem table[] = {
 #  define KNOW_LIB64_DIRS 1
 #elif defined(__i386__) || defined(__x86_64__)
         { "lib64",    0, "usr/lib64\0"
-                         "usr/lib\0",                "ld-linux-x86-64.so.2" },
```

Comment:
no need to change indentation here and other unchanged lines - some editor autoformatting I guess